math/mlsbset/mlsbset.go

```diff
@@ -0,0 +1,122 @@
+// Package mlsbset provides a constant-time exponentiation method with precomputation.
```

Comment:
This seems to be included in the other PR: i'm reviewing it there

Reply:
That is because this PR is on top of the mlsbset PR.